btreeg.go

```diff
@@ -1323,6 +1325,32 @@ func (iter *IterG[T]) Item() T {
 	return iter.item
 }
 
+// Pos returns the position of the current iterator item. Returns -1
+// if the tree is empty or the iterator is at the beginning of the tree.
+func (iter *IterG[T]) Pos() int {
+	if iter.tr == nil {
+		return -1
+	}
+	if !iter.seeked {
+		return -1
+	}
+	count := 0
+	for i, s := range iter.stack {
```

Comment:
this loop is looping through the btree path with O(log N) complexity, is it? If yes, I think need to document this, because I thought this should be O(1) (or rather, O(bree_degree)).

Reply:
Yes. It will approximately follow the height of the tree, which is `O(log N)`. Added it as a comment to the method.